transcom/mymove
# B 19830 ssw text

## [B-19830](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19830) and [B-19843](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19843)

## Summary

[INT PR 1](https://github.com/transcom/mymove/pull/12784)
[INT PR 2](https://github.com/transcom/mymove/pull/12937) - Easiest reference for checking commits

These two stories change the SSW template and it's generation. Firstly, the template will now read "Claimed Expenses" instead of "Claimable Expenses", and secondly, upon generation it will no longer populate the trusted agents section with the releasing and receiving agents information as these are not the same as trusted agents. Instead, it now populates with a message regarding the requirements for setting up a trusted agent with the office manually.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the client application, create and submit a PPM with an AOA request.
2. As a service counselor, review and approve them PPM and Advance request.
3. As either role, download the AOA Packet and ensure the claimed expenses title and trusted agent text have been updated in accordance with the screenshots below.

### Backend


## Screenshots
<img width="642" alt="image" src="https://github.com/transcom/mymove/assets/148906585/5291f919-31a2-469b-93f6-1ee0543bd095">
